Sources/Sentry/SentrySdkInfo.m

```diff
@@ -58,9 +78,36 @@ - (instancetype)initWithDictInternal:(NSDictionary *)dict orDefaults:(SentrySdkI
     return [self initWithName:name andVersion:version];
 }
 
+- (nullable NSString *)getPackageName:(SentryPackageManagerOption)packageManager
+{
+    switch (packageManager) {
+    case SentrySwiftPackage:
+        return @"spm:getsentry/%@";
+    case SentryCocoaPods:
+        return @"cocoapods:getsentry/%@";
+    case SentryCarthage:
+        return @"carthage:getsentry/%@";
+    default:
+        return nil;
+    }
+}
+
 - (NSDictionary<NSString *, id> *)serialize
 {
-    return @{ @"sdk" : @ { @"name" : self.name, @"version" : self.version } };
+    NSMutableDictionary *sdk = @{
+        @"name" : self.name,
+        @"version" : self.version,
+    }
+                                   .mutableCopy;
+    if (self.packageManager != SentryNoPackage) {
+        sdk[@"packages"] = @{
+            @"name" :
+                [NSString stringWithFormat:[self getPackageName:self.packageManager], self.name],
```

Comment:
`h`: `getPackageName` can return `nil` and if it does, we crash here. So either check if it doesn't return `nil` before calling `stringWithFormat` or return a non nil value in `getPackageName`.